staging/src/k8s.io/pod-security-admission/go.mod

```diff
@@ -9,7 +9,6 @@ require (
 	github.com/stretchr/testify v1.7.0
 	k8s.io/api v0.0.0
 	k8s.io/apimachinery v0.0.0
-	k8s.io/apiserver v0.0.0
```

Comment:
Maybe should still update publishing/rules.yaml

Reply:
Unfortunately it's getting added back in https://github.com/kubernetes/kubernetes/pull/103465